delivery/models/project.py

```diff
@@ -154,6 +154,17 @@ def __del__(self):
                 " Check token expiry date (`dds auth info`).")
             raise
 
+
+    def get_dds_version(self):
+        """
+        Get the DDS version been used
+        """
+
+        cmd = ['dds', '--version']
+        version = self._run(cmd) # should return "Data Delivery System, version 2.6.1"
+        return version
```

Comment:
shouldn't the code still strip out the version number, e.g. `2.6.1`, from the returned string? or did you decide to just return the full output?

Reply:
You are write. The only reason I returned the whole string is because I didn't know the `type` returned by actually running the -`run()` method that run` dds --version ` (`ExternalProgramService`)

I was using `subprocess.run` before to run `dds --version` and could decode the output here to convert it to a string that i could split

I tried decoding / just splitting the returned string from the _run() method but that didn't work

**But let me look further into this.**